client/src/components/Ticketing/ticketingmanager/Events/showingInputContainer_deprecated.tsx

```diff
@@ -26,7 +26,7 @@ import DeleteConfirm from './deleteConfirm_depreciated';
 export interface MapPropsToShowingInputContainer {
   showingData: Showing;
   eventinstanceid: number;
-  index: number,
+  index: number;
   handleSetShow: (show: Showing) => void;
   handleDeleteShow: (e: any) => void;
 }
```

Comment:
You can probably remove this file if you find a good spot to put the `toDateStringFormat()` function